composer.json

```diff
@@ -17,7 +17,8 @@
   ],
   "require": {
     "php": "^7.1",
-    "localheinz/json-normalizer": "~0.9.0"
+    "localheinz/json-normalizer": "~0.9.0",
+    "composer/composer": "^1"
```

Comment:
The problem with this is that all we really need is the schema, not `composer/composer` as a dependency. As pointed out earlier, it was removed in #21.

Reply:
I think I will close this PR and use this: https://github.com/localheinz/composer-normalize/issues/80#issuecomment-465044510

What do you think ?